src/python/pants/core/util_rules/external_tool.py

```diff
@@ -102,7 +102,8 @@ def register_options(cls, register):
               - `platform` is one of [{','.join(Platform.__members__.keys())}],
               - `sha256` is the 64-character hex representation of the expected sha256
                 digest of the download file, as emitted by `shasum -a 256`
-              - `length` is the expected length of the download file in bytes
+              - `length` is the expected length of the download file in bytes, as emmitted by 
+                `wc -c`
```

Comment:
I am definitely not a fan of too much help. The shasum above is a _perl_ script! Hopefully folks don't have it and won't install it. Now `wc` is a bit harder to argue against, but `stat -c%s` might be more direct. I know you view this as helpful. I view it as handing the person the fish, and it may even be against their diet.

Reply:
> and it may even be against their diet.

We're not saying "You must use `wc -c`, only offering that that is one way to do this and that it's how Benjy, Stu, and I have been calculating these values from the start. It's meant to give a hint to save time, and if others have a better way of doing it, they can ignore the hint.